vlsi/example.yml

```diff
@@ -120,9 +120,11 @@ par.inputs.physical_only_cells_list:
 
 # SRAM Compiler compiler options
 vlsi.core.sram_generator_tool: "sram_compiler"
-# You should specify a location for the SRAM generator in the tech plugin
-vlsi.core.sram_generator_tool_path: []
+vlsi.core.sram_generator_tool_path: ["hammer/src/hammer-vlsi/technology/asap7"]
 vlsi.core.sram_generator_tool_path_meta: "append"
+# Specify this since we are not using macrocompiler
```

Comment:
Do you mean SRAM compiler? I assumed we would be using MacroCompiler.

Reply:
I think he means he is not using a compiler implementation in macrocompiler and is instead using a cache implementation.